picard/mbxml.py

```diff
@@ -107,7 +126,17 @@ def _relations_to_metadata(relation_lists, m):
             for relation in relation_list.relation:
                 if relation.type == 'performance':
                     performance_to_metadata(relation, m)
+                    if ('work' not in relation.children
+                        or 'id' not in relation.work[0].attribs
+                        ):
+                        log.warning('MBXML: Recording work skipped - no target work MBID present')
```

Comment:
On a general note, I think this and some other warnings do more harm than good because you're testing for the very unlikely case that a recording-work relationship of a recording doesn't have a work associated with it or that work doesn't have an MBID. If we start adding logging warnings on that level, we have a lot of code to add.